x-pack/legacy/plugins/infra/public/components/waffle/node_context_menu.tsx

```diff
@@ -16,14 +16,14 @@ import { findInventoryModel, findInventoryFields } from '../../../common/invento
 import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
 import { InventoryItemType } from '../../../common/inventory_models/types';
 import {
-  ActionMenu,
   Section,
+  SectionLinkProps,
+  ActionMenu,
   SectionTitle,
   SectionSubtitle,
   SectionLinks,
   SectionLink,
-  SectionLinkProps,
-} from './action_menu';
+} from '../../../../../../../x-pack/plugins/observability/public';
```

Comment:
I think you can lose `../x-pack` in this if you want.

Reply:
🤦‍♂